node/executor/src/lib.rs

```diff
@@ -458,6 +460,7 @@ mod tests {
 		(Block { header, extrinsics }.encode(), hash.into())
 	}
 
+	/// TODO: refactor these a bit more.
```

Comment:
remove or add issue number

Reply:
These are self-notes that I always filter and _fix or remove_ before finalizing the PR.